pkgs/development/ruby-modules/gem-config/default.nix

```
@@ -25,7 +25,7 @@
 , cairo, re2, rake, gobject-introspection, gdk-pixbuf, zeromq, czmq, graphicsmagick, libcxx
 , file, libvirt, glib, vips, taglib, libopus, linux-pam, libidn, protobuf, fribidi, harfbuzz
 , bison, flex, pango, python3, patchelf, binutils, freetds, wrapGAppsHook, atk
-, bundler, libsass, libselinux ? null, libsepol ? null, shared-mime-info
+, bundler, libsass, libselinux, libsepol, shared-mime-info, libthai, libdatrie
```

```suggestion
, bundler, libsass, libselinux, libsepol, shared-mime-info, libthai, libdatrie, DarwinTools
```
Please do not use darwin directly as an input.

---

Fixed other gems as well.